x/evm/txs/deliver/deliver.go

```diff
@@ -49,13 +49,15 @@ func (tx *Tx) GetSenderAccount() authexported.Account {
 	return pm.AccountKeeper.GetAccount(infCtx, tx.StateTransition.Sender.Bytes())
 }
 
+// ResetWatcher when panic reset watcher
 func (tx *Tx) ResetWatcher(account authexported.Account) {
```

Comment:
```suggestion
func (tx *Tx) resetWatcher(account authexported.Account) {
```
we would not export this function now